src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -576,6 +603,18 @@ private function formatRouterConfig(array $config): string
         return trim($configAsString);
     }
 
+    /**
+     * @param array<string> $methods
+     */
+    private function formatMethods(array $methods): string
+    {
+        if (!$methods) {
+            $methods = ['ANY'];
+        }
+
+        return implode('|', array_map(fn (string $method): string => '<fg='.self::VERB_COLORS[$method].'>'.$method.'</>', $methods));
```

Comment:
It's possible that $method is not defined in VERB_COLORS (eg TRACE, PURGE, or any custom verb)

Reply:
Should fall back to default now.